src/skills/gotopos/GoToPos.cpp

```diff
@@ -21,6 +21,7 @@ GoToPos::GoToType GoToPos::stringToGoToType(const string &gtt) {
 }
 
 void GoToPos::onInitialize() {
+
```

Comment:
I would rather remove this new line, because it is not useful to clarify the structure of your code